.github/CODEOWNERS

```diff
@@ -1,107 +1,596 @@
+#/<NotInRepo>/            @Drewm3 @avirishuv
+
+# ServiceLabel: %Consumption %Service Attention
+/sdk/consumption/Microsoft.Azure.Management.Consumption/            @ms-premp
+
+# ServiceLabel: %Connected Kubernetes %Service Attention
+#/<NotInRepo>/            @akashkeshari
+
+# ServiceLabel: %Container Instances %Service Attention
+/sdk/containerinstance/Microsoft.Azure.Management.ContainerInstance/            @dkkapur
+
+# ServiceLabel: %Container Registry %Service Attention
+/sdk/containerregistry/            @toddysm @yugangw-MSFT
+
+# ServiceLabel: %Cosmos %Service Attention
+/sdk/cosmosdb/          @Wmengmsft @MehaKaushik @shurd @anfeldma-ms @zfoster @kushagraThapar @southpolesteve
+
+# ServiceLabel: %Cost Management %Service Attention
+/sdk/cost-management/Microsoft.Azure.Management.CostManagement/           @ms-premp @ramaganesan-rg
+
+# ServiceLabel: %Customer Insights %Service Attention
+/sdk/customer-insights/Microsoft.Azure.Management.CustomerInsights/            @shefymk
+
+# ServiceLabel: %CycleCloud %Service Attention
+#/<NotInRepo>/            @adriankjohnson
+
+# ServiceLabel: %Data Bricks %Service Attention
+#/<NotInRepo>/            @yagupta
+
+# ServiceLabel: %DataBox %Service Attention
+/sdk/databox/Microsoft.Azure.Management.DataBox/            @tmvishwajit @matdickson @manuaery @madhurinms
+
+# ServiceLabel: %Data Catalog %Service Attention
+#/<NotInRepo>/            @anilman
+
+# ServiceLabel: %Data Factory %Service Attention
+/sdk/datafactory/Microsoft.Azure.Management.DataFactory/            @shawnxzq @lmy269
+
+# ServiceLabel: %Data Lake %Service Attention
+#/<NotInRepo>/            @sumantmehtams
+
+# ServiceLabel: %Data Lake Storage Gen1 %Service Attention
+#/<NotInRepo>/            @sumantmehtams
+
+# ServiceLabel: %Data Lake Storage Gen2 %Service Attention
+#/<NotInRepo>/            @sumantmehtams
+
+# ServiceLabel: %Data Lake Analytics %Service Attention
+/sdk/datalake-analytics/Microsoft.Azure.Management.DataLake.Analytics/            @idear1203
+
+# ServiceLabel: %Data Lake Store %Service Attention
+/sdk/datalake-store/Microsoft.Azure.Management.DataLake.Store/            @sumantmehtams
+
+# ServiceLabel: %Data Migration %Service Attention
+/sdk/datamigration/Microsoft.Azure.Management.DataMigration/            @rgreenMSFT
+
+# ServiceLabel: %Devtestlab %Service Attention
+/sdk/devtestlabs/Microsoft.Azure.Management.DevTestLabs/            @Tanmayeekamath
+
+# ServiceLabel: %Device Provisioning Service %Service Attention
+/sdk/deviceprovisioningservices/Microsoft.Azure.Management.DeviceProvisioningServices/            @nberdy
 
 # PRLabel: %DigitalTwins
-/sdk/digitaltwins/                              @drwill-ms @timtay-microsoft @abhipsaMisra @vinagesh @azabbasi @bikamani @barustum @jamdavi
+# ServiceLabel: %Digital Twins %Service Attention
+/sdk/digitaltwins/                              @drwill-ms @timtay-microsoft @abhipsaMisra @vinagesh @azabbasi @bikamani @barustum @sourabhguha @inesk-vt @jamdavi
```

Comment:
Why did @sourabhguha and @inesk-vt get added? I'm not against it, but noticing this is a change. They may not want all the PR email.

Reply:
This comes from the internal wiki list. I will add them there for now and if there is too much traffic we can remove.

Reply:
I checked with Saurabh, and he asked me to remove him. I'm checking with Ines.

Is there any benefit to them to be on this list, aside from PR participation?